e2e-tests/serviceless-external-nodes/run

```diff
@@ -33,13 +33,14 @@ kubectl_bin config set-context $(kubectl_bin config current-context) --namespace
 create_namespace $replica_namespace 0
 deploy_operator
 
-kubectl_bin apply \
-	-f "$conf_dir/client.yml" \
-	-f "$test_dir/conf/secrets.yml"
 apply_cluster "$test_dir/conf/external.yml"
 
-wait_pod ${cluster}-rs0-0
-wait_pod ${cluster}-rs0-1
+desc 'create secrets and start client'
+kubectl_bin apply -f $test_dir/conf/secrets.yml
```

Comment:
Why do we apply secrets after creating cluster? Should not they be present when we create custer?